docs/development-api/github.md

```diff
@@ -119,5 +165,23 @@ await GitHub.putIssueComment({
 
 ---
 
+## mapChangedFilesToFileChanges
```

Comment:
Given this is an I/O operation, I wonder if the verb should here be `read` or `load`?

Reply:
Yeah I think that makes sense, when I first started this I thought it was going to be a simple map.